crates/nu-engine/src/documentation.rs

```diff
@@ -68,10 +70,13 @@ fn get_documentation(
 ) -> String {
     // Create ansi colors
     //todo make these configurable -- pull from enginestate.config
-    const G: &str = "\x1b[32m"; // green
-    const C: &str = "\x1b[36m"; // cyan
-                                // was const BB: &str = "\x1b[1;34m"; // bold blue
-    const BB: &str = "\x1b[94m"; // light blue (nobold, should be bolding the *names*)
+    let g: String = get_ansi_color_for_component_or_default(engine_state, "header", "\x1b[32m"); // default: green
+
+    let c: String = get_ansi_color_for_component_or_default(engine_state, "row_index", "\x1b[36m"); // default: cyan
+                                                                                                    // was const bb: &str = "\x1b[1;34m"; // bold blue
+    let bb: String =
+        get_ansi_color_for_component_or_default(engine_state, "foreground", "\x1b[94m"); // default: light blue (nobold, should be bolding the *names*)
```

Comment:
As they are now not just color short hands maybe it makes sense to expand those variables to more readable names.